ananse/peakpredictor.py

```diff
@@ -579,7 +707,7 @@ def predict_proba(self, factor=None, motifs=None, jaccard_cutoff=0.0):
 
         Predict binding probability for either a TF (factor) or a set of
         motifs. Prediction will be based on the data that been loaded,
-        either ATAC-seq or H3K27ac data or both.
+        either ATAC-seq or H3K27ac data or both or CAGE bidirectional regions.
```

Comment:
"either ATAC-seq, H3K27ac ChIP-seq, ATAC-seq and H3K27ac ChIP-seq or bidirectional CAGE-seq regions."